src/plugins/packages/index.js

```diff
@@ -97,8 +97,10 @@ function loadPackages() {
 			return;
 		}
 
+		const version = packageInfo.version;
 		packageInfo = packageInfo.thelounge;
 		packageInfo.packageName = packageName;
+		packageInfo.version = version;
```

Comment:
You read version from `packageInfo.version` already, I don't think this or line 100 are needed?

Reply:
as noted in the PR description, I saved the version into the packageInfo since then others get access to it via the public client.
Not something that is 100% needed now, but I liked the idea of having the version next to the name in that packageInfo object (what you define under "thelounge" in your package.json)

Reply:
Ah whoops, I missed the `packageInfo = packageInfo.thelounge;`.